fdbserver/StorageMetrics.actor.cpp

```diff
@@ -292,6 +292,10 @@ void StorageServerMetrics::splitMetrics(SplitMetricsRequest req) const {
 			if (key == req.keys.end)
 				break;
 			reply.splits.push_back_deep(reply.splits.arena(), key);
+			if (reply.splits.size() > SERVER_KNOBS->SPLIT_METRICS_MAX_ROWS) {
```

Comment:
This file doesn't seem to define any ACTOR functions, so it could be renamed as `StorageMetrics.cpp`.

Reply:
cc @sfc-gh-xwang what do you think?